### PR TITLE
Team 2 jay vith stepper 2

### PR DESCRIPTION
**Pull request by JayVith**

### CODE REVIEW

File: src/pages/welcome/index.js [hunk 1]
```diff
@@ -8,15 +8,17 @@ import "./style.css";
 const Welcome = () => {
   const { onCreateProfile } = useAuth();
 
+  const [profile, setProfile] = useState({
+    firstName: "",
+    lastName: "",
+    githubUsername: "",
+    bio: "",
+    password: "",
```
Comment (procrastian) [inline src/pages/welcome/index.js:16]: Maybe swap password for something more contact info related? Password is already assigned and verified when they sign up.

File: src/pages/welcome/index.js [hunk 2]
```diff
@@ -30,7 +32,10 @@ const Welcome = () => {
       profile.firstName,
       profile.lastName,
       profile.githubUsername,
-      profile.bio
+      profile.bio,
```
Comment (dearshrewdwit) [inline src/pages/welcome/index.js:35]: What's the reasoning behind providing these args to onCreateProfile?

the method signature for this func declared in src/context/auth.js shows: 

```js
  const handleCreateProfile = async (firstName, lastName, githubUrl, bio) => {
  
  }
  ```

Comment (JayVith) [reply src/pages/welcome/index.js:35]: Thought they maybe needed for the profile page and the fields team 4 were creating, but if that's not the case can simply remove them

File: src/pages/welcome/index.js (hunk 1 above)
Comment (JayVith) [reply src/pages/welcome/index.js:16]: Thought that the field is being used for the profile page and will be editable there which team 4 is handling

**Review by Shylan21 — APPROVED** [review]
Looking good

**Pull request closed without merging**